docs/pipelines/multisample/shiva.md

```diff
@@ -83,6 +83,9 @@ unifiedgenotyper:
 At this moment the following variant callers can be used
 
 ##### Germline
+
+When doing variantcalling mostly germline is used. This will detect variants based on the assumption that there is a fixed number of alleles. Mostly the default used is a ploidy of 2. When this assumption does not hold for your data somatic variantcalling can be a better solution.
```

Comment:
`variantcalling` -> `variant calling`. 
`Mostly` -> `Most often,`
There should be a comma between `data` and `somatic` in the third sentence.